ci-operator/step-registry/openstack/test/manifests/openstack-test-manifests-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: openstack-test-manifests
+  from: openstack-installer
+  commands: openstack-test-manifests-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: OS_CLOUD
+      documentation: Name of cloud to use from ${CLUSTER_PROFILE_DIR}/clouds.yaml file
+    - name: OPENSTACK_EXTERNAL_NETWORK
+      documentation: Name of external network to attach FIPs to.
+    - name: OPENSTACK_COMPUTE_FLAVOR
```

Comment:
Perhaps a good time to change `OPENSTACK_COMPUTE_FLAVOR` to something a little bit less confusing like `OPENSTACK_INSTANCE_FLAVOR` or `OPENSTACK_DEFAULT_FLAVOR`